files/ja/web/html/microformats/index.md

```diff
@@ -161,7 +161,7 @@ Example h-entry as a blog post:
   <p><span class="p-author h-card">
     <a href="https://quickthoughts.jgregorymcverry.com/profile/jgmac1106" ><img class="u-photo" src="https://quickthoughts.jgregorymcverry.com/file/2d6c9cfed7ac8e849f492b5bc7e6a630/thumb.jpg"/></a>
     <a class="p-name u-url" href="https://quickthoughts.jgregorymcverry.com/profile/jgmac1106">Greg McVerry</a></span>
-     Replied to <a class="u-in-reply-to" href="https://developer.mozilla.org/en-US/docs/Web/HTML/microformats">a post on
+     Replied to <a class="u-in-reply-to" href="https://developer.mozilla.org/ja/docs/Web/HTML/microformats">a post on
```

Comment:
The base language of this document is English. Do you think it should be a link to `ja` even?

Reply:
Other locales have also changed this to point to their own locale, so I think this change is good!

Reply:
Is there a reason something else "other locale also..."?

Reply:
I don't think that it matters too much as this is just example code anyways, but I'll let you decide!